docs/data/joy/components/table/table.md

```diff
@@ -0,0 +1,272 @@
+
+```js
+<Table stripe="3n">
+```
+
+:::
+
+### Hover
+
+To highlight a row of the table body when hover on it, set `hoverAxis` prop to true.
+
+{{"demo": "TableHover.js"}}
+
+### Border
+
+Use `borderAxis` prop to control the border appearance.
+
+{{"demo": "TableBorder.js"}}
+
+<details>
+<summary><b>Adding custom values</b></summary>
+
+Customize the table theme based on `borderAxis` prop using [`extendTheme()`](/joy-ui/customization/themed-components/#change-styles-based-on-props) function.
+
+```js
+import { CssVarsProvider, extendTheme } from '@mui/joy/styles';
+
+const theme = extendTheme({
+  components: {
+    JoyTable: {
+      styleOverrides: {
+        root: ({ ownerState }) => ({
+          ...(ownerState.borderAxis === 'header' && {
+            '& thead th:not([colspan])': {
+              border: '2px solid var(--TableCell-borderColor)',
+            },
+          }),
+        });
+      }
+    }
+  }
+})
+
+<CssVarsProvider theme={theme}>…</CssVarsProvider>
+```
+
+For **TypeScript**, you have to add the new values via module augmentation:
+
+```ts
+// this could be any file that's included in your tsconfig.json
+declare module '@mui/joy/Table' {
+  interface TablePropsBorderAxisOverrides {
+    header: true;
+  }
+}
+```
+
+</details>
+
+### Sticky header
+
+Set `stickyHeader` prop to true. The table must be a child of a fixed-height element with overflow `auto` (or `scroll`).
+
+Using [Sheet](/joy-ui/react-sheet/) component is recommended for wrapping a table. See [usage with Sheet](#usage-with-sheet) to learn more.
+
+{{"demo": "TableStickyHeader.js"}}
+
+### Caption
+
+Place `<caption>` as the first child of the table.
+
+{{"demo": "TableCaption.js"}}
+
+:::success
+To display caption at the bottom of the table, set [caption side](https://developer.mozilla.org/en-US/docs/Web/CSS/caption-side) to `bottom`:
+
+```js
+<Table sx={{ captionSide: 'bottom' }}>
+```
+
+:::
+
+### Footer
+
+Use [`<tfoot>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/tfoot) to show the table footer.
+
+{{"demo": "TableFooter.js"}}
+
+### Row head
+
+Set `scope="row"` to `th` elements inside `tbody` to apply the same style as column headers.
+
+{{"demo": "TableRowHead.js"}}
+
+### Row and column span
+
+Use `rowSpan` and `columnSpan` to expand a cell to multiple rows or columns.
+
+{{"demo": "TableRowColumnSpan.js"}}
+
+:::info
+**Limitation**: the CSS cannot figure out how apply borders on every corner without duplication.
+
+There are two workarounds:
+
+1. Manually remove the right border for a specific cell:
+
+   ```js
+   // In this example, `Canceled` header cell must not have a border on the right.
+   <th style={{ borderRightWidth: 0 }}>Canceled</th>
+   ```
+
+2. Set Table's border collapse to `collapse`:
+
+   ```js
+   <Table sx={{ borderColapse: 'collapse' }}>
+   // However, this will change the behavior when `stickyHeader` is true.
+   // The border bottom of the sticky headers will no longer sticked.
```

Comment:
```suggestion
   // Note that this will change the behavior when `stickyHeader` is true:
   // the bottom border of the sticky header will not "stick".
```